make/hotspot/lib/CompileJvm.gmk

```diff
@@ -100,7 +100,7 @@ endif
 
 DISABLED_WARNINGS_xlc := tautological-compare shift-negative-value
 
-DISABLED_WARNINGS_microsoft := 4624 4244 4291 4146 4127
+DISABLED_WARNINGS_microsoft := 4624 4244 4291 4146 4127 4722
```

Comment:
I forgot to mention this in the PR summary.  New warning suppression for
4722: 'function' : destructor never returns, potential memory leak
We have various destructors calling ShouldNotCallThis, ShouldNotReachHere,
or the like, which triggers this warning.  It might be possible to deal with
some (but not all) of those by deleting the destructor.  The warning doesn't
seem all that useful to us, so I just suppressed it.